File: src/main/java/lt/ltrp/InventoryEntityImpl.java
package lt.ltrp;

import lt.ltrp.item.Inventory;

/**
 * @author Bebras
 *         2015.11.29.
 */
public class InventoryEntityImpl extends NamedEntityImpl implements InventoryEntity {

    private Inventory inventory;

    public InventoryEntityImpl(int id, String name, Inventory inventory) {
        super(id, name);
        this.inventory = inventory;
    }

    public InventoryEntityImpl() {

    }

    public Inventory getInventory() {
        return inventory;
    }

    public void setInventory(Inventory inventory) {
        this.inventory = inventory;
    }
}

File: shoebill_plugins/LtrpApi/src/main/java/lt/ltrp/object/impl/InventoryEntityImpl.kt
package lt.ltrp.`object`.impl;

import lt.ltrp.`object`.Inventory;
import lt.ltrp.`object`.InventoryEntity;

/**
 * @author Bebras
 *         2015.11.29.
 */
abstract class InventoryEntityImpl(uuid: Int, name: String, override var inventory: Inventory?) : NamedEntityImpl(uuid, name), InventoryEntity {

    constructor(uuid: Int, name: String): this(uuid, name, null) {

    }

}
